docs/conf.py

```diff
@@ -115,7 +119,7 @@
     # an arbitrary url.
     'navbar_links': [
         ("Blog", "https://medium.com/pangeo", True),
-        ("Forum", "https://discourse.pangeo.io", True),
+        ("Discourse Forum", "https://discourse.pangeo.io", True),
```

Comment:
I'm not sure that adding 'Discourse' is really needed in the navbar.

Reply:
I agree that a longer name might be less useful, but Aimee and I both mentioned that we were confused about this terminology during our first few Pangeo website visits years ago. In my experience, most Pangeo folks refer to it as "Discourse" and in initially I didn't know that "Discourse" was a web forum. 

Maybe we can get some others to weight in here, but I'd still vote for "Discourse Forum" in the navbar for extra clarity.